Makefile.in

```diff
@@ -136,6 +136,16 @@ report-binutils: report-binutils-@default_target@
 .PHONY: report-gdb
 report-gdb: report-gdb-@default_target@
 
+stamps/check-write-permission:
+	touch $(INSTALL_DIR)/.test || \
+		(echo "Sorry, you don't have permission to write" \
+		 "'$(INSTALL_DIR)', use --prefix to specify" \
+		 "another path, or use 'sudo make' if you *REALLY* want" \
+		 "install into '$(INSTALL_DIR)'" && exit 1)
+	rm $(INSTALL_DIR)/.test
+	touch $@
+	mkdir -p $(dir $@) && touch $@
```

Comment:
You have two touch commands, I think you only need the second one.

Reply:
Fixed.